WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -220,7 +220,32 @@
     <% end %>
 
     <%= f.input :currency_code, collection: Money::Currency.table.values, label_method: lambda { |c| c[:name]  }, value_method: lambda { |c| c[:iso_code] }  %>
-    <%= f.input :base_entry_fee_lowest_denomination %>
+    <%= f.input :base_entry_fee_lowest_denomination, input_html: { type: "text" } %>
```

Comment:
Nice idea, I'll combine it with your comment on setting up an onchange listener below.